IPNWB_Utils.ipf

```diff
@@ -816,11 +816,18 @@ End
 /// Note: This is typically located at the location of the IPNWB program ipf files.
 ///       @c FunctionPath is not threadsafe
 Function/S SpecificationsDiscLocation()
-	string path = FunctionPath("")
-	string folder = GetFolder(path)
 
-	ASSERT_TS(FolderExists(folder), "Buggy return value with " + path + ", " + folder)
-	return folder
+	string path
+
+	PathInfo IPNWB_DiscLocation
+	if(V_flag)
+		return s_path
+	endif
+
+	path = GetFolder(FunctionPath(""))
+	NewPath/Q/O IPNWB_DiscLocation, path
```

Comment:
You create a new symbolic path but you do not check if the path exists. What is the purpose of the symbolic path in this context?

Reply:
`FunctionPath` in an IM was initially thought of as being buggy. And that symbolic path was meant to cache the result of `FunctionPath`.

But the real bug was that the try/catch statement in the caller of SpecDiscLocation did not clear the RTE so `AbortONRTE` triggered on a lingering RTE.